tests/ftests/075-pybindings-cgroup_compare_cgroup.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgroup_compare_cgroup() test using the python bindings
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli, Mode
+from libcgroup import Cgroup, Version
+import consts
+import ftests
+import sys
+import os
+
+CGNAME = '075cgcompare/compare'
+CONTROLLERS = ['cpu', 'memory', 'io', 'pids']
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    if Cgroup.cgroup_mode() != Mode.CGROUP_MODE_UNIFIED:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires the unified cgroup v2 hierarchy'
+        return result, cause
+
+    return result, cause
+
+
+def setup(config):
+    CgroupCli.create(config, CONTROLLERS, CGNAME)
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    #
+    # Test 1 - Compare matching empty cgroup instances
+    #
+    cgempty1 = Cgroup(CGNAME, Version.CGROUP_V2)
+    cgempty2 = Cgroup(CGNAME, Version.CGROUP_V2)
+
+    if cgempty1 != cgempty2:
+        result = consts.TEST_FAILED
+        tmp_cause = 'Empty cgroups do not match'
+        cause = '\n'.join(filter(None, [cause, tmp_cause]))
```

Comment:
This will be the first of the warnings/errors, it can be simplified to `cause = 'Empty cgroups do not match'` and remove `tmp_cause`